hack/release/operator/make_helm.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
this is about preparing a helm chart, so what about calling the file `make_chart.sh` ?

Reply:
Sure why not.